_data/site_menu.yml

```diff
@@ -178,66 +161,99 @@ administration:
           - title: IBM Cloud Object Storage Providers
             path: "/docs/reference/latest/managing_providers/storage_providers/ibm_cloud_object_storage_managers.html"
             desc: "Managing IBM Cloud Object Storage Providers"
-
     - title: Provisioning Virtual Machines and Hosts
       path: "/docs/reference/latest/provisioning_virtual_machines_and_hosts/index.html"
       desc: "Provisioning, workload management, and orchestration in ManageIQ"
-
+    - title: Creating a Service in for Virtual Machine Provisioning
+      path: "/docs/reference/latest/creating_a_service_for_vm_provisioning/index.html"
+      desc: "How to create a service for virtual machine provisioning"
+    - title: Service Provisioning Using an Orchestration Template
+      path: "/docs/reference/latest/service_provisioning_using_an_orchestration_template/index.html"
+      desc: "How to create orchestration template to deploy services"
```

Comment:
missing an article: `"How to create an orchestration template to deploy services"